tests/backup/backup_sse_test.go

```diff
@@ -261,6 +268,43 @@ var _ = Describe("{CreateBackupAndRestoreForAllCombinationsOfSSES3AndDenyPolicy}
 				}
 				wg.Wait()
 			})
+
+			Step("Validating restores with sc and ns mapping", func() {
+				log.InfoD("Validating restores with sc and ns mapping")
+				ctx, err := backup.GetAdminCtxFromSecret()
+				log.FailOnError(err, "Fetching px-central-admin ctx")
+				appContextsToBackups := FilterAppContextsByNamespace(scheduledAppContexts, []string{bkpNamespaces[midpoint]})
```

Comment:
Please refer to https://github.com/portworx/torpedo/pull/1968/files#r1425311525

Reply:
done